docs/code.rst

```diff
@@ -63,6 +63,8 @@ Operator API
         PythonOperator,
         S3KeySensor,
         S3ToHiveTransfer,
+        SlackAPIOperator,
```

Comment:
I think sphinx won't be able to find this one if it's not added to `operators/__init__.py`